my-react-app/src/app/components/baseComponents/Board/index.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+import Square from '../Square';
+import './styles.scss';
+
+class Board extends React.Component {
+  renderSquare = i => <Square value={this.props.squares[i]} onClick={() => this.props.onClick(i)} />;
+
+  render() {
+    return (
+      <React.Fragment>
```

Comment:
Import `Fragment` directly from the `react` package.

`import React,{Fragment} from 'react';`